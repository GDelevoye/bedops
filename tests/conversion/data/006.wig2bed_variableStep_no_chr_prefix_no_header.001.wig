variableStep chrom=2
300701 12.5
300702 12.5
300703 12.5
300704 12.5
300705 12.5 
