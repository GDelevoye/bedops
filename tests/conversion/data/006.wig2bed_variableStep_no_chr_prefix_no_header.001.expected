2	300700	300701	id-1	12.500000
2	300701	300702	id-2	12.500000
2	300702	300703	id-3	12.500000
2	300703	300704	id-4	12.500000
2	300704	300705	id-5	12.500000
